README.md

```diff
@@ -249,6 +249,11 @@ The `OTNetworkTest()` constructor includes the following parameters:
     in OpenTok.js). For more information, please check the 
     [IP Proxy Documentation](https://tokbox.com/developer/guides/ip-proxy/).
 
+  * `scalableVideo` (Boolean) -- (Optional) Whether to use
+    [scalable video](https://tokbox.com/developer/guides/scalable-video/)
+    (`true`, the default) or not (`false`). Disabling scalable video
+    was added in OpenTok.js version 2.24.7.
```

Comment:
Based on the code below, I think the default is `false`, no?

Reply:
Isn't the default setting for the `scalableVideo` Publisher option in OpenTok.js `true`? In the code below, if you don't pass in an option, it uses the OpenTok.js default. Please confirm @v-kpheng .

Reply:
@jeffswartz  - Is the plan to have the Precall Tool use this option and make the default false in the Tool but leave the default in the module as unset which will pick up the SDK default of true?

Reply:
@rhainer I am awaiting clarification on the requirements, but I think just setting `scalableVideo` to `false` is the best bet (without adding an option to set it, at least for now).